src/SupportMessage/SupportMessage.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC } from 'react'
+import styled, { css } from 'styled-components'
+import { lighten } from 'polished'
+
+import { Box } from '../Box'
+import { Icon } from '../Icon'
+import { theme } from '../theme'
+
+export type SupportMessageProps = {
+  className?: string
+  children: string
+  type: 'info' | 'alert' | 'warning'
```

Comment:
I'm guessing we need to align first with design if we talk about `type` or something else?

Reply:
I mean, we don't really care much on how design call them since in Figma you won't really see a name, you just infer from the look of the component which type/variation we need to have.

Reply:
We can see it on Figma as well, it's currently named state

![Screenshot 2022-05-26 at 09 43 52](https://user-images.githubusercontent.com/14129033/170452736-e0a2f49f-e937-4224-a5d4-3d2990dfa56e.png)

Reply:
I matched them to icons used otherwise naming doesn't make sense and too disruptive to change Icon names directly

Reply:
I think Mickael was talking about the prop name not the values. I guess we can go with `state` if that's what's used in Figma

Reply:
AH! I see i misunderstood. Ok i can change to state

Reply:
From what I've seen, they use state for everything currently but I think we can talk with design so they use `type` or something more explicit instead
`state` is too general as a term